bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -99,6 +99,21 @@ spec:
       name: resource-dir
     - mountPath: /etc/kubernetes/static-pod-certs
       name: cert-dir
+  - name: kube-apiserver-insecure-readyz-REVISION
+    image: ${OPERATOR_IMAGE}
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command: ["cluster-kube-apiserver-operator", "insecure-readyz"]
+    args:
+    - --insecure-port=6080
```

Comment:
Any reason why we picked 6080,
this is quite separate/far from 6443, something like 6440 to give room and still close enough...

Not important, feel free to ignore we can program lbs to use anything :)

Reply:
6000 + 443 vs. +80